### PR TITLE
[CHORE] adding website container

### PR DESCRIPTION
**Pull request by nicolastakashi**
Adding website container.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,5 +1,20 @@
 version: '3.9'
 services:
+  website:
+    build:
+      context: .
+      dockerfile: ./website/Dockerfile
```
Comment (Nexucis) [inline docker-compose.yaml:6]: does it mean to update the website we will need to remove the local image and then rebuild it ?

Comment (nicolastakashi) [reply docker-compose.yaml:6]: Actually we can just run docker-compose up --build

File: website/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+RUN npm install --ignore-scripts
+
+RUN hugo --minify --gc --cleanDestinationDir
+
+FROM hugomods/hugo:nginx
+
+# Copy the generated files to keep the image as small as possible.
+COPY --from=builder /src/public /site
+
+RUN ls -la /site
```
Comment (Nexucis) [inline website/Dockerfile:31]: Are you sure you want to run this command when building the image ? That's a debug layer for me, probably we should remove it for "production".

Comment (nicolastakashi) [reply website/Dockerfile:31]: Ups!
YOu're right, I'll remove

**Review by Nexucis — APPROVED** [review]
nice ! let's try that !

**Pull request merged**